src/Package/MSTest.Sdk/Sdk/Runner/Common.targets

```diff
@@ -26,4 +26,9 @@
     <MicrosoftTestingExtensionsCodeCoverageVersion Condition=" '$(MicrosoftTestingExtensionsCodeCoverageVersion)' == '' " >$(MicrosoftTestingExtensionsCodeCoverageVersion)</MicrosoftTestingExtensionsCodeCoverageVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePlaywright Condition=" '$(EnablePlaywright)' == '' ">false</EnablePlaywright>
+    <MicrosoftPlaywrightVersion Condition=" '$(MicrosoftPlaywrightVersion)' == '' ">$(MicrosoftPlaywrightVersion)</MicrosoftPlaywrightVersion>
```

Comment:
Where is `MicrosoftPlaywrightVersion` default version set?

Reply:
https://github.com/microsoft/testfx/blob/main/src/Package/MSTest.Sdk/Sdk/Sdk.props.template

Reply:
Thanks, fixed.